api/client/network.go

```diff
@@ -113,6 +113,7 @@ func (cli *DockerCli) CmdNetworkRm(args ...string) error {
 			status = 1
 			continue
 		}
+		fmt.Fprintf(cli.out, "%s\n", net)
```

Comment:
I understand the differences, but [Fprintf](https://golang.org/src/fmt/print.go?h=doPrintf#L1101) does a lot more handling under the hood that's not needed here, so was wondering if we should just use println (given that we don't use any real formatting here). Sorry, just thinking out loud